### DISCUSSION TITLE
Add requirement for DirectX to be installed to readme

### DISCUSSION THREAD

**Discussion by k0pp0** [emoose/OutRun2006Tweaks#249] [Suggestions]
### Describe the feature you'd like to see added

just a pointer - on a fresh windows 11 build I had to download DirectX runtimes on top of the VC redistributables.

here's the link: https://download.microsoft.com/download/1/7/1/1718CCC4-6315-4D8E-9543-8E28A4E18C4C/dxwebsetup.exe

**Comment by Outrunner24** [comment]
Well...Duh, it's basically an unwritten rule for all Windows games, especially the older ones 😉

**Comment by k0pp0** [comment]
Yep, its unwritten, so helping people who don't know is the plan..
